src/hooks/useCallLog.ts

```diff
@@ -0,0 +1,105 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { useCallback, useEffect } from 'react';
+
+import { useAppSelector } from '@/redux/hooks';
+import type { ICallLog } from '@/types/calllog.d';
+
+type SortOption = 'newest' | 'oldest';
```

Comment:
1. Move hooks to inbox folder
2. All axios + redux
3. Scroll bar and Pagination